src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_ScrollViewer.cs

```diff
@@ -1244,7 +1245,10 @@ public async Task When_TouchScroll_Then_NestedElementReceivePointerEvents()
 			var sutLocation = sut.GetAbsoluteBounds().GetLocation();
 			finger.Drag(sutLocation.Offset(5, 480), sutLocation.Offset(5, 5));
 
-			events.Should().BeEquivalentTo("enter", "pressed", "release", "exited");
+			// The expected proper sequence should really be "Enter", "Pressed", "PointerCaptureLost", "Exited"
+			// This will need proper equivalent of Windows' Direct Manipulation.
+			// For now, it's important to not get Released as it can cause a click on the nested element while it's being scrolled.
```

Comment:
The impacts on the pointer events of the "Direct Manipulation" is more a side effect / hack than a feature ... I would not consider that as a desired behavior as anyway it's impacted by the `ManipulationMode` set on sub-elements. And unless plans have changed, the WinUI team was planning to remove such inconsistency in in pointer events in the new `ScrollView` (not validated yet).

```suggestion
			// The expected proper sequence when all sub-elements are ManipulationMode=System should be "Enter", "Pressed", "PointerCaptureLost", "Exited"
			// This is caused by the Windows' Direct Manipulation.
			// The important thing is to not get Released as it can cause a click on the nested element while it's being scrolled.
```